### PR TITLE
New dashboard

### PR DESCRIPTION
**Pull request by adrianob**

### CODE REVIEW

**Comment by lerrua** [comment]
:beers:

File: app/assets/javascripts/app/app.js [hunk 1]
```diff
@@ -25,6 +26,10 @@ var App = window.App = Skull.View.extend({
     }
   },
 
+  toggleEdit: function(event){
+    $("#edit-menu-items").slideToggle( "slow" );
```
Comment (vicnicius) [inline app/assets/javascripts/app/app.js]: Could we implement a CSS solution here? Do the transition on CSS, only changing the class, just like on the toggleNav function.

File: config/initializers/simple_form.rb [hunk 2]
```diff
@@ -46,7 +46,7 @@
     b.optional :readonly
 
     b.wrapper :label_wrapper, tag: 'div' do |ba|
-      ba.use :label_text, wrap_with: { tag: :label, class: "field-label fontweight-semibold" }
+      ba.use :label_text, wrap_with: { tag: :label, class: "field-label fontweight-semibold fontsize-base" }
```
Comment (vicnicius) [inline config/initializers/simple_form.rb]: Have you guys checked for unwanted layout changes on other forms?

Comment (adrianob) [inline config/initializers/simple_form.rb]: Yes, this wrapper is only being used on dashboard pages.

**Pull request merged**